web/server/vue-cli/src/assets/userguide/userguide.md

```diff
@@ -14,429 +14,759 @@ on GitHub</a>
-- **Diff** - In this column you can select two runs (*baseline* and
-*compared to*) which will be compared to each other by click on the *Diff*
-button.
-- **Name** - Name of the run.
-- **Number of unresolved reports** - Number of non unique reports excluding
+This page contains the analysis runs available on the server under the selected
+product.
+
+Each run has the following fields/columns:
+- `Delete`: in the first column of the run list table you can select multiple
+runs which will be removed by clicking on the *DELETE* button. For more
+information [see](#delete-runs).
+- `Unique name` which identifies a run. If an already existing run name is
+used during the storage, the run will be updated: a new history event will
+be created for the run and the detection status will be changed for each
+report.
+  - `Version tag` (optional): a unique identifier for this individual store of
+results in the run's history.
+  - `Description` (optional): a custom textual description to be shown alongside
+the run. You can find more information how to see the description
+[here](#show-description).
+  - `Check command` which was used to analyze your project. You can find more
+  information how to see the check command [here](#show-check-command).
+  - `Detection status counts`: show the number of reports distributed by
+  detection statuses.
+- `Number of unresolved reports`: number of non unique reports excluding
 *Resolved*, *False positive* and *Intentional* reports.
-- **Storage date** - Storage date of the runs.
-- **Analysis duration** - Duration of the analysis.
-- **Check command** - By clicking on *Show* text the check command will be shown
-in a pop-up window.
-- **Detection status** - Detection statuses are calculated based on the detection
-status values from the previous store where the reports are stored again with
-the same run name. When storing the results of a run from scratch then each
-report will have detection status of *New*.
-- **Version tag** - Latest version tag of the run.
-- **Delete** - In this column you can select multiple runs which will be removed
-by clicking on the *Delete* button.
-
-## <a name="userguide-filter-runs"></a> Filter runs
+- `Analyzer statistics`: information about the analysis. You can find more
+information how to see the analysis statistics
+[here](#show-analyzer-statistics).
+- `Latest storage date`: latest storage/update date of the run.
+- `Analysis duration`: the length of the analysis.
+- `CodeChecker version` which was used during the analysis.
+- `Diff`: in this column you can select two runs (*baseline* and
+*compared to*) which will be compared to each other by click on the *DIFF*
+button. For more information [see](#compare-runs).
+
+### Show description
+Description is a custom textual description to be shown alongside the run and
+given by the `CodeChecker store` command:
+```sh
+CodeChecker store \
+  --name master_xerces \
+  --description "This is my demo run." \
```

Comment:
On the example image some different description is visible, not what was set here in the command example.